tensorboard/plugins/debugger/README.md

```diff
@@ -0,0 +1,144 @@
+# The Debugger Dashboard
+
+The debugger dashboard offers a graphical user interface for the [TensorFlow debugger](https://www.tensorflow.org/versions/r1.1/programmers_guide/debugger). For instance, this dashboard
+
+* enables users to pause execution at specified ops or continue execution for some number of steps.
+* visualizes values of tensors.
+* associates tensors with specific lines in python code.
+
+![The Debugger Dashboard](images/debugger_intro.png)
+
+# Setup
+
+## Instrumenting Model Logic
```

Comment:
I would move this section after the next two sections, i.e. "## Start TensorBoard with the `debugger_port` flag." and "## Navigating to TensorBoard". That would be the most easy-to-follow order of doing things, because the user will be able to see the initial dialog and how it automatically disappears when the tf.Session connects to tensorboard.